NAMESPACE

```diff
@@ -79,19 +79,15 @@ export(intersect_maf)
 export(liftover_bedpe)
 export(maf_to_custom_track)
 export(make_igv_snapshot)
-export(plot_multi_timepoint)
```

Comment:
Not sure what happened with these 4 functions that were discontinued - but trust it was discussed somewhere. Just want to double-check with you it is intentional

Reply:
Yes, these functions are still in GAMBLR, I just removed the @export for them, since they are all either in draft mode or/and are not yet doing anything. I thought this was the best approach since it does not make sense for a user to have access to functions that are not doing anything. Especially since this package is now open to the public. I also talked this over with Ryan when he was at BC Cancer a week ago, and he agreed. Thanks for checking!

Reply:
Thanks for confirming!